mayastor/src/bdev/dev/nvmx/controller.rs

```diff
@@ -201,33 +201,39 @@ impl<'a> NvmeController<'a> {
         if ns.is_null() {
             warn!(
                 "{} no namespaces reported by the NVMe controller",
-                self.get_name()
+                self.name
             );
         }
 
         self.inner.as_mut().unwrap().namespaces =
             vec![Arc::new(NvmeNamespace::from_ptr(ns))]
     }
 
+    /// Get controller state.
+    pub fn get_state(&self) -> NvmeControllerState {
```

Comment:
nit; would just `state()` not be nicer?

Reply:
That would be idiomatic: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter